src/main.rs

```diff
@@ -81,6 +81,7 @@ lazy_static! {
         "https://github.com/andoriyu/uclicious", // FIXME: CI hack. the crate has a higher count, but we don't refresh.
         "https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi", // marketplace link , but also has enough stars
         "https://github.com/TraceMachina/nativelink", // 1.4k stars, probably broken because @palfrey now works for them...
+        "https://app.repoflow.io/repoflow-public/package/f429fabf-6289-49c2-acd9-791b39eac746", // added per discussion in the RepoFlow pull request: https://github.com/rust-unofficial/awesome-rust/pull/2054
```

Comment:
Url should be https://www.repoflow.io as that's the one in the README. The package link should be in the comment as that's the justification for adding it